### PR TITLE
Display AWS Inspector results

### PR DESCRIPTION
**Pull request by adamnfish**
## What does this change?

Adds AWS Inspector results for each account to the app.

### Homepage
homepage

### All accounts
all-inspector-results

### Single account
account-inspector-results

## Will this require CloudFormation and/or updates to the AWS StackSet?
Yes, there is a change to `watched-account.template.yaml` to enable AWS Inspector calls.

**StackSet update required prior to deploy**

### CODE REVIEW

File: hq/app/logic/InspectorResults.scala [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import model.InspectorAssessmentRun
+import org.joda.time.DateTime
+import utils.attempt.FailedAttempt
+
+import scala.collection.JavaConverters._
+
+
+object InspectorResults {
+  private val tagMatch = "[\\w\\-_\\.]"
+  val RunNameMatch = s"AWSInspection-($tagMatch+)-($tagMatch+)-($tagMatch+)-[\\d]+".r
```
Comment (adamnfish) [inline hq/app/logic/InspectorResults.scala]: This will change if the Lambda changes its format

Comment (shtukas) [reply hq/app/logic/InspectorResults.scala]: +1

Comment (adamnfish) [reply hq/app/logic/InspectorResults.scala]: We can change this separately and exclude non-Lambda runs at the same time

File: hq/app/logic/InspectorResults.scala [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import model.InspectorAssessmentRun
+import org.joda.time.DateTime
+import utils.attempt.FailedAttempt
+
+import scala.collection.JavaConverters._
+
+
+object InspectorResults {
+  private val tagMatch = "[\\w\\-_\\.]"
+  val RunNameMatch = s"AWSInspection-($tagMatch+)--($tagMatch+)--($tagMatch+)--[\\d]+".r
```
Comment (shtukas) [inline hq/app/logic/InspectorResults.scala]: Updated this with a new commit @adamnfish @aug24

**Comment by aug24** [comment]
+1 with latest changes.

**Comment by adamnfish** [comment]
Watched account update is out, so let's YOLO this out.

**Pull request merged**